Jenkinsfile

```diff
@@ -29,63 +29,6 @@ pipeline {
       }
     }
 
-    stage('Build Docker Images'){
-      steps {
-        withCredentials([string(credentialsId: 'sidekiq-enterprise-license', variable: 'BUNDLE_ENTERPRISE__CONTRIBSYS__COM')]) {
-          sh 'env=$RAILS_ENV make build'
-        }
-      }
-    }
```

Comment:
It might be good to still leave this step since we aren't building docker in GHA yet.

Reply:
The built image here was being used for testing and linting yes? That's why I removed it, I didn't see any other stage that would be using it. If I'm wrong and there is a stage that requires it, let me know.

Reply:
Right. I'm not saying the step is required but it would be good for a process to continue to run and report to us if a `make build` fails. We haven't moved that to GHA yet so until we do, we should probably continue testing it in Jenkins.

Reply:
Perhaps the Build AMI stage already does that and would be sufficient?